boot/grubfm/rules/iso/e2b_test.sh

```diff
@@ -1,17 +1,12 @@
 source ${prefix}/func.sh;
```

Comment:
Is this still compatible with the use of (hd0,1) type partition naming?

Reply:
Isnt the first section redundant? The second section will always be executed anyway.

Reply:
does ^hd work??  shouldn't it be  '^\(hd*.
![image](https://user-images.githubusercontent.com/1469350/81053546-280b1700-8ebd-11ea-9eee-fd590b8368f2.png)